Makefile

```diff
@@ -22,4 +22,4 @@ console:
 	docker run --network=host $(CONTAINER_ENV_VARS) $(INTERACTIVE_SESSION) $(CMD)
 
 test:
-	$(MAKE) console USER=root MIX_ENV=test CMD="mix do local.hex --force, local.rebar --force, deps.get, test"
+	$(MAKE) console USER=root MIX_ENV=test CMD="mix do local.hex --force, local.rebar --force, deps.get, test $(FILE)"
```

Comment:
Added this to make our Vim plugin work with `make test`. We have a shortcut `space+t` that runs `make test FILE=<path>:<line>` and allows us to run only the test under the cursor.